### PR TITLE
Delete load balancer operation.

### PR DESCRIPTION
**Pull request by derekolk**
Implemented openstack delete load balancer operation.

### CODE REVIEW

File: clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy [hunk 1]
```diff
@@ -282,6 +284,42 @@ abstract class OpenstackClientProvider {
   }
 
   /**
+   * Remove load balancer pool.
+   * @param region
```
Comment (lwander) [inline clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy]: While we're normally not great a generating code docs, it feels strange to see these `param` annotations with descriptions of the variables being passed in.

**Comment by lwander** [comment]
LGTM, any objections to merging?

**Comment by ashleykasim** [comment]
Looks good to me as well, @lwander go ahead and merge

**Comment by ashleykasim** [comment]
https://github.com/spinnaker/spinnaker/issues/915

**Pull request merged**